lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -76,7 +76,7 @@ module.exports = [
               {resourceType: 'script', requestCount: 3, transferSize: '55000±1000'},
               {resourceType: 'image', requestCount: 2, transferSize: '28000±1000'},
               {resourceType: 'document', requestCount: 1, transferSize: '2200±100'},
-              {resourceType: 'other', requestCount: 1, transferSize: '1000±50'},
+              {resourceType: 'other', requestCount: 1, transferSize: '1030±100'},
```

Comment:
this number is 1036 locally, 1059 in travis, 50 seemed a little too harsh for something that doesn't really matter anyway